data/map.txt

```diff
@@ -30694,3 +30700,2835 @@ planet Zug
+	government "Gegno Vi"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet Enbi
+	landscape land/fog0
+	description "A cold, mountainous world on the very edge of the binary system's habitable area, Enbi maintains the massive shipyards of the Gegno Vi, using raw resources gathered from the nearby volcanic planet to churn out hull after hull for warships. Many large canyons decorate the surface and are used as weapon test sites, and in one of the largest canyons lies a shipyard several times larger and older-looking than the rest."
+	description "	On closer inspection, the method that created the canyons does not seem entirely natural, and there are slight glints of light shimmering down in the deepest regions. How these crevices appeared is a mystery, but it is clear that the Vi had no hand in creating them."
+	spaceport "Rows of alternating towers stretch far ahead and behind the one your spacecraft is stationed at, and a thick layer of fog acts as a sort of floor connecting them. Every now and then, the fog breaks, revealing the large shipyards down below either side of you, followed by a new row of towers."
+	spaceport "	Off in the distance of the breaking fog is one vast gray structure, almost taking up your field of view. The closer towards the mass, the more buildings there seems to be, and it's unlikely you could get anywhere near there without becoming a threat."
+	spaceport "	If you stare too long, the nearby guards gesture more aggressively towards you, suggesting you keep your attention to yourself."
+	government "Gegno Vi"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet "Oup Je"
+	landscape land/canyon4
+	description "The many beveling formations here provide the Gegno Scin with a rich amount of materials for their cities and ships. Lower down below are small woodlands that seem to be stripped down one way or another, with few living areas scattered about. It is apparent that this planet has been worn down over time, both to natural and unnatural causes."
+	spaceport "The Scin here have different gear than those leaving and entering the spaceport, suggesting they have a particular place on the surface rather than outside of the planet, probably some of the miners. They seem quite interested in you compared to other Gegno being that a majority of their days involve seeing mostly materials and machinery."
+	government "Gegno Scin"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet "Enn Bue"
+	landscape land/canyon01
+	description "Lengthy oval-like shipyards speckle the surface of this otherwise inhospiable desert planet, home to most of the Gegno Scin's ship production. A majority of wealthy materials come from the neighboring gas giant's moons, however the strong gravitational pull occasionally generates natural disasters that ravage the surface. Despite all of this, the Scin remain viligant and efficient in their craftworks."
```

Comment:
```suggestion
	description "Lengthy oval-like shipyards speckle the surface of this otherwise inhospitable desert planet, home to most of the Gegno Scin's ship production. A majority of wealthy materials come from the neighboring gas giant's moons; however, the strong gravitational pull occasionally generates natural disasters that ravage the surface. Despite all of this, the Scin remain vigilant and efficient in their craftworks."
```